u19_pipeline/recording.py

```diff
@@ -0,0 +1,182 @@
+import datajoint as dj
+from u19_pipeline import lab, task, subject
+
+schema = dj.schema('u19_recording')
+
+from u19_pipeline import acquisition, subject
+
+
+@schema
+class RecordingModality(dj.Lookup):
+     definition = """
+     recording_modality:        varchar(64)          # modalities for recording (ephys, imaging, video_recording, etc.) 
+     ---
+     modality_description:      varchar(255)         # description for the modality
+     root_direcory:             varchar(255)         # root directory where that modality is stored (e.g. ephys = /braininit/Data/eletrophysiology)
+     recording_file_extensions: blob                 # file extensions specific for this modality
```

Comment:
```suggestion
     recording_file_extensions: varchar(16)                 # file extensions specific for this modality
```

Reply:
I used blob here because we could have a list (that could even grow) of file extensions for an specific modality.
For instance I use this in the GUI to look for folders with "valid" recordings

Reply:
Sounds great.  Thanks.